PayloadPkg/Include/FwuCbContext.h

```diff
@@ -0,0 +1,117 @@
+/** @file
+
+  Copyright (c) 2017 - 2019, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef __FWUCBCONTEXT_H__
+#define __FWUCBCONTEXT_H__
+
+#include <RegAccess.h>
+
+// prototypes takens from MdePkg\Include\Library\MemoryAllocationLib.h
+typedef VOID *(*FWU_ALLOCATE_POOL_CB)(UINTN AllocationSize);
```

Comment:
let us copy the function parameter comments over to here in the same way as other API such as FWU_HECI_READ_MESSAGE_CB

Reply:
this file is not required, removed this file